twilio/http/orgs_token_manager.py

```diff
@@ -0,0 +1,42 @@
+from twilio.base.version import Version
+from twilio.http.token_manager import TokenManager
+from twilio.rest.preview_iam.organizations.token import TokenList
+
+
+class OrgTokenManager(TokenManager):
+    """
+    Orgs Token Manager
+    """
+
+    def __init__(
+        self,
+        grant_type: str,
+        client_id: str,
+        client_secret: str,
```

Comment:
Can this constructor be used when we need to init using, grant type, client id and refresh token instead of 
grant type, client id and client secret combination

Reply:
yes, these data points will be used [here](https://github.com/twilio/twilio-python/blob/8f026a0c9637097e38b900ac225c8bd93a6a9e4a/twilio/http/orgs_token_manager.py#L31), which will then be used normally to create token or use existing one.